app/views/static_pages/maths_and_physics.html.erb

```diff
@@ -0,0 +1,52 @@
+<% content_for(:page_title, t("maths_and_physics.policy_name")) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= policy_service_name("maths-and-physics") %>
+    </h1>
+
+    <p class="govuk-body">
+      Claim £2,000 after tax if you:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>are a maths or physics teacher or teach maths or physics sometimes</li>
+      <li>
+        have a degree or teaching qualification specialising in maths or physics —
+        <%= link_to "check which qualifications are eligible", "https://www.gov.uk/guidance/apply-for-mathematics-and-physics-teacher-retention-payments#opportinity-area", class: "govuk-link" %>
+      </li>
+      <li>
+        teach at a state-funded secondary school in certain places —
+        <%= link_to "check which places are eligible", "https://www.gov.uk/guidance/apply-for-mathematics-and-physics-teacher-retention-payments#regional-eligibility-criteria", class: "govuk-link" %>
+      </li>
+      <li>completed your initial teacher training (ITT) course in or after the 2013 to 2014 academic year</li>
+    </ul>
+
+    <p class="govuk-body">
+      The application takes around 15 to 45 minutes to complete.
+    </p>
+
+    <h2 class="govuk-heading-m">Before you apply</h2>
+
+    <p class="govuk-body">You'll need:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>your National Insurance Number</li>
+      <li>your bank account details</li>
+      <li>
+        your 7-digit teacher reference number – you can get this from your
+        school, the certificate you got when you qualified, or from the
+        <%= link_to "teacher qualifications helpdesk", "https://www.gov.uk/guidance/individual-teacher-records-information-for-teachers#contact", class: "govuk-link" %>
+      </li>
+      <li>the date you completed your initial teacher training</li>
+      <li>
+        sign in details for ‘Verify’ — the government’s secure way of proving
+        who you are, if you do not have a Verify account, you’ll need your
+        passport or photocard driving licence to sign up
+      </li>
+    </ul>
+
+    <%= link_to "Start now", new_claim_path("maths-and-physics"), class: "govuk-button govuk-button--start", role: "button" %>
```

Comment:
Does this need the arrow? I think it's inline now so the styles don't add it.

https://design-system.service.gov.uk/components/button/

It could also do with `data-module="govuk-button"` like the other comment.

Reply:
I figured it wasn't worth using the arrow here given the [actual start pages we end up with](https://www.gov.uk/guidance/teachers-claim-back-your-student-loan-repayments) don't have them.